src/main/java/com/epam/processor/DataProcessor.java

```diff
@@ -71,7 +87,10 @@ public RoadAccident getAccidentByIndex7(String index){
      * @return
      */
     public Multimap<String, String> getAccidentIdsGroupedByAuthority7(){
-        return null;
+        Multimap<String, String> resultMap = ArrayListMultimap.create();
+        roadAccidentList.stream()
+                .forEach(accident -> resultMap.put(accident.getDistrictAuthority(), accident.getAccidentId()));
+        return resultMap;
```

Comment:
I actually wanted you to use java 7 with Guava, but this is really nice combination of streaming and guava multimap. Good job